### PR TITLE
Pull Request #677: Github link removed from Logo

### PR DESCRIPTION
**Pull request by iknoorjobs**

### CODE REVIEW

File: howtoplay.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       <div class="header">
         <img class="play-button" src="images/docs/play-button.png" alt="Play" />
         <h1 class="site_title centered lang" data-href="how">HOW TO PLAY</h1>
-        <a href="https://github.com/fossasia/labyrinth">
+        <a href="">
```
Comment (Ploypiti) [inline howtoplay.html]: Maybe you should link this to `index.html` or `#` instead.

Comment (duttaditya18) [reply howtoplay.html]: Yes. Point it to `#`. That'll be good.

**Comment by niccokunzmann** [comment]
Hi @iknoorjobs, thanks for your pull request. In order to keep track of what happens over time, could you tell me, how this improves the game for you? If every change is motivated like this, we end up building a better game step by step.

**Comment by harsh-jindal** [comment]
@iknoorjobs Please make the above requested changes. And while making a new pull request please follow the pull request template. You need to mention the Issue Number at least, otherwise it is difficult  to find the issue fixed.

**Comment by niccokunzmann** [comment]
This looks like it. Thanks!

**Pull request merged**